WLC/benchmark.py

```diff
@@ -27,18 +27,34 @@ def benchmark_file(file_name):
     image = Preprocessor().process(picture)
     code = image.get_code().lower()
     difference = editdistance.eval("".join(code.split()), "".join(expected_code.split()))
-    accuracy = round(100 - (difference * 100 / len(expected_code)))
+
+    length = len("".join(expected_code.split()))
+    accuracy = round(100 - (difference * 100 / length))
+
     LOGGER.info('Accuracy: {}%, File: {}'.format(accuracy, file_name))
+    return accuracy, length
 
 
 def run_benchmarks():
     LOGGER.info('=== Whiteboard Live Coding Benchmarking ===')
     LOGGER.info('Uses Levenshtein distance to calculate the difference and then uses that to calculate accuracy.')
     LOGGER.info('')
 
+    total_accuracy = 0
+    total_length = 0
+
     for i in range(1, 6):
-        benchmark_file('assets/examples/images/example_{}.png'.format(i))
+        accuracy, length = benchmark_file('assets/examples/images/example_{}.png'.format(i))
+        total_accuracy += accuracy * length
+        total_length += length
+
+    overall_accuracy = round(total_accuracy / total_length)
+
+    LOGGER.info('')
```

Comment:
I know it's for good formatting but it doesn't look nice.

Reply:
Its not the nicest but will keep for formatting.